### PR TITLE
Feature: Detalles viaje, crear rutina de conductor y borrar rutinas

### PR DESCRIPTION
**Pull request by ghost**
El único problema es que no está pillando bien las coordenadas Leaflet, pero se arreglará seguramente ahora. Crear rutina de conductor no tiene validadores, también se hará proximamente.

### CODE REVIEW

File: bugalink-frontend/pages/ride/[id]/details.tsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+              <p className="text-sm font-normal">Plazas libres</p>
+              <p className="text-xl font-bold">
+                {trip.driver_routine.available_seats}
+              </p>
+            </div>
+          </div>
+          {requested === 'false' && (
+            <div className="flex justify-center">
+              <Link
+                href="/ride/V1StGXR8_Z5jdHi6B-myT/detailsTwo"
```
Comment (andrsdt) [inline bugalink-frontend/pages/ride/[id]/details.tsx:173]: This part of code (appearing when query param of `details?requested=false`) has not been adapted to the new structure without `detailsTwo`. It should be named something like "PAGAR" and with an href linking to the ride payment. Take as an example the behavior in the deleted `detailsTwo.tsx`

File: bugalink-frontend/pages/search/result.tsx [hunk 2]
```diff
@@ -317,33 +317,33 @@ export default function SearchResults() {
...
+                rating={trip.rating}
+                name={trip.name}
+                gender={trip.gender}
+                avatar={trip.avatar}
+                origin={trip.origin}
+                destination={trip.destination}
+                date={trip.date}
+                price={trip.price}
+                className="rounded-md bg-white outline outline-1 outline-light-gray"
+                href={NEXT_ROUTES.RIDE_DETAILS_ONE(trip.id)}
```
Comment (andrsdt) [inline bugalink-frontend/pages/search/result.tsx:343]: `RIDE_DETAILS_ONE` should be renamed to `RIDE_DETAILS` now that we don't have `detailsTwo`

File: bugalink-frontend/cypress/e2e/navigation.cy.ts [hunk 3]
```diff
@@ -49,23 +49,23 @@ describe('Navigation tests', () => {
...
+  // /ride/<rideId>/details
   it('should navigate to ride details one page', () => {
     cy.visit('/');
     cy.get('[data-cy="ride-details"]').first().click();
-    cy.location('pathname').should('match', /^\/ride\/[\w-]+\/detailsOne$/);
+    cy.location('pathname').should('match', /^\/ride\/[\w-]+\/details$/);
   });
 
   // /ride/<rideId>/detailsTwo
   it('should navigate to ride details two page', () => {
```
Comment (andrsdt) [inline bugalink-frontend/cypress/e2e/navigation.cy.ts:60]: This test should be changed such that we click on something like "PAGAR" and it takes us to the page for selecting payment methods for the ride

File: bugalink-frontend/constants/nextRoutes.ts [hunk 4]
```diff
@@ -10,7 +10,7 @@ const NEXT_ROUTES = {
   MY_ROUTINES: `/routines`,
   RIDE_HISTORY: '/history',
   RIDE_DETAILS_ONE: (rideId, requested = false) =>
-    `/ride/${rideId}/detailsOne?requested=${requested}`,
+    `/ride/${rideId}/details?requested=${requested}`,
   RIDE_DETAILS_TWO: (rideId) => `/ride/${rideId}/detailsTwo`,
```
Comment (andrsdt) [inline bugalink-frontend/constants/nextRoutes.ts:14]: This should be deleted

**Review by andrsdt — CHANGES_REQUESTED** [review]
There are some changes that have to be made before merging, check the comments. Apart from that, awesome job!! 🥳 💯

File: bugalink-frontend/constants/nextRoutes.ts (hunk 4 above)
Comment (alvvazort) [reply bugalink-frontend/constants/nextRoutes.ts]: All references of details two has been deleted in a new commit

File: bugalink-frontend/cypress/e2e/navigation.cy.ts (hunk 3 above)
Comment (alvvazort) [reply bugalink-frontend/cypress/e2e/navigation.cy.ts:60]: Done

File: bugalink-frontend/pages/search/result.tsx (hunk 2 above)
Comment (alvvazort) [reply bugalink-frontend/pages/search/result.tsx]: Refactor done

File: bugalink-frontend/pages/ride/[id]/details.tsx (hunk 1 above)
Comment (alvvazort) [reply bugalink-frontend/pages/ride/[id]/details.tsx]: image

**Review by andrsdt — APPROVED** [review]
Everything working now, we still have to fix the timezone problem but we will merge for now to add the new changes!

**Pull request merged**